lib/data/dto/expenses/expense_dto.dart

```diff
@@ -1,17 +1,18 @@
 import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:trip_tally/data/dto/expenses/expense_category_dto.dart';
+import 'package:trip_tally/data/dto/price/price_dto.dart';
 import 'package:trip_tally/domain/entities/expenses/expense_entity.dart';
 
 part 'expense_dto.freezed.dart';
+
 part 'expense_dto.g.dart';
 
 @freezed
 class ExpenseDto with _$ExpenseDto {
   const factory ExpenseDto({
     required String name,
     required String date,
-    required double amount,
-    required String currency,
+    required PriceDto price,
     required String tripId,
     required String categoryId,
     ExpenseCategoryDto? category,
```

Comment:
is 'category' optional? I suppose in form to adding expenses , the field with category is required. Or maybe,  we should have default category